docs/src/main/asciidoc/spring-cloud-gateway.adoc

```diff
@@ -971,6 +971,24 @@ The Gateway can be configured to create routes based on services registered with
 
 To enable this, set `spring.cloud.gateway.discovery.locator.enabled=true` and make sure a `DiscoveryClient` implementation is on the classpath and enabled (such as Netflix Eureka, Consul or Zookeeper).
 
+== CORS Configuration
+
+The gateway can be configured to control CORS behavior. The "global" CORS configuration is a map of URL patterns to https://docs.spring.io/spring/docs/5.0.x/javadoc-api/org/springframework/web/cors/CorsConfiguration.html[Spring Framework `CorsConfiguration`]. 
+
+.application.yml
+[source,yaml]
+----
+spring:
```

Comment:
Could you add another example that demonstrates a more real world configuration rather than permitting everything?